pkg/services/navtree/navtreeimpl/applinks.go

```diff
@@ -233,6 +233,7 @@ func (s *ServiceImpl) readNavigationSettings() {
 		"grafana-oncall-app":               {SectionID: navtree.NavIDAlertsAndIncidents, SortWeight: 1},
 		"grafana-incident-app":             {SectionID: navtree.NavIDAlertsAndIncidents, SortWeight: 2},
 		"grafana-ml-app":                   {SectionID: navtree.NavIDAlertsAndIncidents, SortWeight: 3},
+		"grafana-cloud-link-app":           {SectionID: navtree.NavIDCfg},
```

Comment:
Maybe I'm missing something but if it is related to `moving`, as you said in the description, shouldn´t it be deleted from somewhere else? Or is just adding the link? 🤔

Reply:
here (https://github.com/grafana/grafana/blob/ash/58213/pkg/services/navtree/navtreeimpl/applinks.go#L176-L182) it looks at the `navigationAppConfig` and decides what the `sectionID` should be. if there's nothing in `navigationAppConfig`, it gets added to `navtree.NavIDApps`. otherwise it gets added to where you specify in the config. so we don't need to delete it from `Apps` because it never gets added there anymore 🙂

Reply:
Good to know @ashharrison90, thanks for the explanation.
So, in that case, LGTM 👍